examples/the-link.masm

```diff
@@ -0,0 +1,127 @@
+what can I say except h
+
+sneak 100 ecx
+upgrade
+eax is great, but I want 116
+what can I say except al
+upvote ecx
+corporate needs you to find the difference between ecx and 2
+fuck go back
+
+they're the same picture
+
+parry 4 you filthy casual eax
+what can I say except al
+
+what can I say except s
+
+what can I say except 58
+
+parry 65 you filthy casual eax
+what can I say except al
+what can I say except al
+stonks eax
+
+sneak 100 eax
+parry 135 you filthy casual eax
+what can I say except al
+parry 10 you filthy casual eax
+what can I say except al
+stonks eax
+
+upvote eax
+upvote eax
+upvote eax
+upvote eax
+upvote eax
+upvote eax
+what can I say except al
+stonks eax
+downvote eax
+what can I say except al
+not stonks eax
+what can I say except al
+
+eax is great, but I want 101
+stonks eax
+upvote eax
+bitconeeeeeeect eax 4294967290
+what can I say except al
+not stonks eax
+what can I say except al
+
+what can I say except .
+
+what can I say except c
+not stonks eax
+what can I say except al
+
+bitconeeeeeeect eax 253
+what can I say except al
+
+not stonks eax
+what can I say except al
+
+what can I say except w
+what can I say except 97
+
+sneak 100 ebx
+parry 652 you filthy casual ebx
+what can I say except bl
+
+edx is great, but I want ebx
+parry 17 you filthy casual edx
+what can I say except dl
+
+parry 12 you filthy casual ebx
+what can I say except bl
+
+what can I say except ?
+
+what can I say except 118
+
+parry 242 you filthy casual eax
+what can I say except al
+
+eax is great, but I want 100
+what can I say except al
+parry 19 you filthy casual eax
+what can I say except al
+stonks eax
+
+sneak 100 eax or draw 25
+stonks eax or draw 25
+parry 12 you filthy casual eax
+what can I say except al
+
+stonks eax
+what can I say except 4
+not stonks eax
+what can I say except al
+what can I say except 9
+
+not stonks eax
+stonks eax
+
+upvote eax
+upvote eax
+upvote eax
+upvote eax
+upvote eax
+upvote eax
+what can I say except al
+
+downvote ebx
+what can I say except bl
+
+
+
+parry 15 you filthy casual ebx
+what can I say except bl
+
+what can I say except c
+
+not stonks eax
+what can I say except al
+
+what can I say except \n
```

Comment:
I didn't fall for it this time ;)
But nice example there, I'm impressed!

Reply:
:D